src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -146,7 +147,8 @@ public void printPdf(Claim claim, List<Printable> documents) {
             return document.template.getBytes();
         }
 
-        return pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
+        byte[] html = pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
```

Comment:
have you tested this change anywhere?

Reply:
I have reverted back to old code as it was removed by mistake

Reply:
We need tests